__tests__/basics.test.ts

```diff
@@ -1,14 +1,11 @@
 import * as httpm from '../_out';
 import * as path from 'path';
-import * as am from '../_out/auth';
```

Comment:
are you going to revert this change?  would like to finish up this PR ...

Reply:
This import isn't used, it was removed completely

Reply:
ok